src/main/kotlin/no/nav/hjelpemidler/soknad/mottak/util/AutoGenerateDocumentTitle.kt

```diff
@@ -8,10 +8,21 @@ import no.nav.hjelpemidler.soknad.mottak.client.hmdb.HjelpemiddeldatabaseClient
 import no.nav.hjelpemidler.soknad.mottak.util.slack.PostToSlack
 
 object AutoGenerateDocumentTitle {
-    private val defaultTitle = "Søknad om hjelpemidler"
+    private val defaultTitleSoknad = "Søknad om hjelpemidler"
+    private val defaultTitleBestilling = "Bestilling av hjelpemidler"
+    private val bestillingPrefix = "Bestilling av:"
+    private val soknadPrefix = "Soknad om:"
 
     fun generateTitle(soknad: JsonNode): String {
-        if (Configuration.application.profile == Profile.LOCAL) return defaultTitle
+        val behovsmeldingsType = soknad["behovsmeldingType"]?.asText()
```

Comment:
Siden `soknad` er en JsonNode fra Jackson ville jeg forventet at `soknad["behovsmeldingType"]` returnerer en NullNode i steden for null. Så usikker på om ?-tegnet er nødvendig her. Så har man forskjellen mellom asText() og textValue(). asText som du har brukt her vil returnere en string-verdi for alle verdier, inkludert null. Så tror "null" kan settes til behovsmeldingsType her.

Reply:
Nå som jeg har lest neste linje i koden så slår det meg at det ikke nødvendigvis gjør noe siden vi bare sjekker om `behovsmeldingsType == "BESTILLING"`. Så dette funker nok det. Ser det er 2 tester for de forskjellige verdiene under, er det tester hvor nehovsmeldingType feltet ikke er satt i soknad? Hadde vært fint før en eventuell tidlig prod.setting.

Reply:
?-operatoren håndterer hvertfall de tilfellene der søknaden ikke inneholder behovsmeldingType, der det ellers ville blitt et nullpointerexception ved videre kall. Men enig i at textValue er bedre å bruke her, så switcher over. La også til en test for det caset du beskrev. Sjekker at ingen exceptions kastes og at dokumentbeskrivelsen blir "Søknad om..."